src/ensembl/src/content/app/custom-download/components/checkbox-with-radios/CheckboxWithRadios.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Checkbox from 'src/shared/checkbox/Checkbox';
+import Radio, { radioOptions } from 'src/shared/radio/Radio';
+
+import styles from './CheckboxWithRadios.scss';
+
+type Props = {
+  radioOptions: radioOptions;
+  label: string;
+  selectedOption: string;
+  onChange: (selectedOption: string) => void;
+};
+
+const CheckboxWithRadios = (props: Props) => {
+  const [checkedStatus, setCheckedStatus] = useState(false);
+
+  useEffect(() => {
+    if (props.selectedOption) {
+      setCheckedStatus(true);
+    }
+  }, []);
+
+  const handleCheckboxOnChange = useCallback(
+    (checkedStatus: boolean) => {
+      setCheckedStatus(checkedStatus);
+      props.onChange('');
+    },
+    [props.selectedOption]
+  );
+
+  const handleOnChange = useCallback(
+    (selectedOption: string) => {
+      props.onChange(selectedOption);
+    },
+    [props.selectedOption]
+  );
+
+  return (
+    <>
```

Comment:
no need for a fragment when there's a single child

Reply:
That's done.